templates/components/_footer.html.twig

```diff
@@ -0,0 +1,124 @@
+<footer class="background--lighter space--30-0">
+    <div class="l__wrapper">
+        <div class="l__row l__row--h-stretch b__nudge--bottom-large l__tablet--col l__mobile--col-center">
+            <a href="{{ path('homepage') }}" class="link--no-decor">
+                {{ include('components/logo-mini-em.html.twig') }}
+            </a>
+            <ul class="list__links list__links--row list__links--no-decor list__links--svgs l__hide--on-tablet">
+                <li class="head">Suivez La République En Marche !</li>
+                <li><a href="https://www.facebook.com/EnMarche/" target="_blank" rel="noopener noreferrer"><i class="fa fa-facebook-square"></i></a></li>
+                <li><a href="https://twitter.com/enmarchefr" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="https://www.instagram.com/enmarchefr/" target="_blank" rel="noopener noreferrer"><i class="fa fa-instagram"></i></a></li>
+                <li><a href="https://www.youtube.com/channel/UCJw8np695wqWOaKVhFjkRyg" target="_blank" rel="noopener noreferrer"><i class="fa fa-youtube-square"></i></a></li>
+                <li><a href="https://medium.com/@enmarchefr" target="_blank" rel="noopener noreferrer"><i class="fa fa-medium"></i></a></li>
+                <li><a href="https://www.linkedin.com/company/en-marche" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin-square"></i></a></li>
+                <li><a href="https://www.snapchat.com/add/enmarchefr" target="_blank" rel="noopener noreferrer"><i class="fa fa-snapchat-ghost"></i></a></li>
+            </ul>
+        </div>
+        <div class="l__row l__row--h-stretch l__row--top b__nudge--bottom-large l__row--wrap l__mobile--col">
+            <ul class="list__links list__links--col list__links--no-decor">
+                <li>
+                    <h4>
+                        <a href="{{ path('page_le_mouvement') }}" class="text--bold text--medium-small">Le Mouvement</a>
+                    </h4>
+                </li>
+                <li>
+                    <a href="{{ path('page_le_mouvement') }}">Qui sommes-nous ?</a>
```

Comment:
Same as above, intended?

Reply:
I think so, @marineb can you confirm that point ?

Reply:
@marineb